server/routes/api/comments/comments.ts

```diff
@@ -72,8 +74,17 @@ router.post(
     authorize(user, "read", comment);
     authorize(user, "read", document);
 
+    let anchorText: string | undefined;
+
+    if (includeAnchorText) {
+      anchorText = ProsemirrorHelper.getAnchorTextForComment(
```

Comment:
Move this inside `presentComment(comment, { includeAnchorText })` and I think it'll cleanup nicely. Would just need to ensure `comment.document` is available